src/fidesctl/api/ctl/sql_models.py

```diff
@@ -85,6 +86,46 @@ def process_result_value(self, value, dialect):
         pass
 
 
+class ClassificationDetail(Base):
+    """
+    The SQL model for a classification instance
+    """
+
+    __tablename__ = "cls_classification_detail"
+    instance_id = Column(String(255))
+    status = Column(Text)
```

Comment:
Provide a more explicit name for what status this is for (i.e. `review_status`)

Reply:
As an alternative solution here, maybe we can provide better context of which status this means (detail field level) with the API responses regardless of column name here